From 3ef22ceaeb702cc2d70170b882d15684ed0de713 Mon Sep 17 00:00:00 2001
From: Santiago Rossi <santiago_rho_psi@hotmail.com>
Date: Mon, 16 Apr 2012 21:32:44 -0300
Subject: [PATCH 27/40] =?UTF-8?q?4.4.8=20Cuenca=20de=20Sholazar=20:=20=C2=BF?=
 =?UTF-8?q?Ya=20has=20vuelto=3F?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Added missing spawn of Mistcaller Soo-gan, thanks to @oceanor
---
 .../reanemu/2012_04_16_109_world_back_so_soon.sql  |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)
 create mode 100644 sql/updates/world/reanemu/2012_04_16_109_world_back_so_soon.sql

diff --git a/sql/updates/world/reanemu/2012_04_16_109_world_back_so_soon.sql b/sql/updates/world/reanemu/2012_04_16_109_world_back_so_soon.sql
new file mode 100644
index 0000000..b1f99fa
--- /dev/null
+++ b/sql/updates/world/reanemu/2012_04_16_109_world_back_so_soon.sql
@@ -0,0 +1,7 @@
+DELETE FROM `creature` WHERE `id`=28114;
+INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`,`MovementType`) VALUES
+(NULL,28114,571,1,1,0,0,6165,5092.98,-97.377174,0.750492,300,0,0,117700,3809,0);
+
+DELETE FROM `creature_template_addon` WHERE `entry`=28114;
+INSERT INTO `creature_template_addon` (`entry`,`path_id`,`mount`,`bytes1`,`bytes2`,`emote`,`auras`) VALUES
+(28114,0,0,65536,1,0,'51589');
\ No newline at end of file
-- 
1.7.9.msysgit.0

